project.go

```diff
@@ -0,0 +1,391 @@
+// Copyright 2016 EF CTX. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/tsuru/gnuflag"
+	"github.com/tsuru/tsuru/cmd"
+	tsuruerrors "github.com/tsuru/tsuru/errors"
+)
+
+type projectCreate struct {
+	fs       *gnuflag.FlagSet
+	name     string
+	platform string
+	team     string
+	plan     string
```

Comment:
It makes sense for us to have different routers, I think. Each associated with a different ELB, I suppose. And that would be the main reason for me. I think reducing memory for dev environments makes sense as well, but it is more optional.